lib/uvm/bridge.browser.js

```diff
@@ -45,7 +45,7 @@ var _ = require('lodash'),
                         emit(e.data.__emit_uvm);
                 });
             }(__uvm_dispatch, "${id}"));
-            __uvm_emit('["load.${id}"]');
+            __uvm_emit('${Flatted.stringify(['load.' + id])}');
```

Comment:
Just to be clear, this is not intended to be a template string, right?

Reply:
You’re right Kunal. It was supposed to be an IPC message sent on first load with a simple load event. Originally, the message structure was crafted by hand since the structure was a simple array with one time ID as parameter.

As such, it should have earlier been done via JSON.stringify. But because of its simplicity, I hadn’t done it that way. Now with Flatted, I had to make it machine generated.